cmd/govim/testdata/scenario_default/complete_watched.txt

```diff
@@ -4,13 +4,12 @@
 vim ex 'e main.go'
 cp const.go.orig const.go
 errlogmatch '&protocol\.DidChangeWatchedFilesParams\{\n\S+:\s+Changes: \{\n\S+:\s+\{URI:"file://'$WORK/const.go'", Type:1\}'
-vim ex 'call cursor(6,16)'
-vim ex 'call feedkeys(\"i\\<C-X>\\<C-O>\\<C-N>\\<C-N>\\<ESC>\", \"x\")'
+vim ex 'call cursor(6,17)'
+vim ex 'call feedkeys(\"i\\<C-X>\\<C-O>\\<C-N>\\<ESC>\", \"x\")'
 vim ex 'w'
 cmp main.go main.go.golden
 
 # Assert that we have received no error (Type: 1) or warning (Type: 2) log messages
-# Disabled pending resolution to https://github.com/golang/go/issues/34103
```

Comment:
This line was accidentally deleted by the looks of things. Hence the build failures.

Reply:
Oh, is this line required? I deleted it because I figured the test wasn't disabled but I probably misunderstood the purpose of the line. I'll put it back.